### DISCUSSION TITLE
bulk edit panel is too small and not resizable

### DISCUSSION THREAD

**Discussion by g-industrialist** [Kong/insomnia#5597] [Ideas]
When I bulk edit my query values, I find the panel is too small and I cannot adjust the size.
![image](https://user-images.githubusercontent.com/40657526/210049351-01176179-c087-4906-8426-dac03851288b.png)

Version: Insomnia 2022.7.0
Build date: 12/15/2022
OS: Windows_NT x64 10.0.19042
Electron: 22.0.0
Node: 16.17.1
V8: 10.8.168.20-electron.0
Architecture: x64

**Comment by Wazbat** [comment]
I can confirm I'm still having this same issue on windows 10

**Comment by TalesRK** [comment]
Same on ubuntu 20.04.6 LTS

![image](https://github.com/Kong/insomnia/assets/37679931/8e9038df-031f-4008-b562-2c9045c8aa81)

**Comment by TalesRK** [comment]
It was fixed on Feb 22, 2023: https://github.com/Kong/insomnia/pull/5795
Just update to the latest version.
